### PR TITLE
Feature/6

### PR DESCRIPTION
**Pull request by YumeAyasaki**
In this PR, I have:

- Checkout from the feature/8 branch...
- Make a footer component...
- Add icons to the public folder...
- Include said component to the _app.js file, thus making it a theme component...
- And IDK if this's needed or not, I make the dropdown menu (Navbar) can be turned off (?) by clicking outside of said dropdown...

Note: The same note as the feature/7, about the icons. And one more, do we need to make this component, like, universal? There's some web that has the Navbar highlight the section they're currently in, but considering our current design, I don't know if we should do that or not...
And I have yet done the linking things...
localhost_3000_Test - Cá nhân 1 - Microsoft​ Edge 9_16_2022 12_04_28 PM
localhost_3000_Test - Cá nhân 1 - Microsoft​ Edge 9_16_2022 12_04_10 PM
localhost_3000_Test - Cá nhân 1 - Microsoft​ Edge 9_16_2022 12_04_19 PM

### CODE REVIEW

File: components/theme/Navbar.jsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    }
+  })
+
+  return (
+    <div className={styles.header}>
+        {/* Navbar on phone */}
+        {
+          clicked && 
+          <div className={styles.navBarPhone} ref={dropdown}>
+            <h3 className={styles.title}>Vườn hoa sứ</h3>
```
Comment (PaulHo0501) [inline components/theme/Navbar.jsx]: Change to "Vườn Hoa Sứ"

File: components/theme/Navbar.jsx [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+            {menuSection.map(section => <h5 className={styles.item} key={section.id}>{section.context}</h5>)}
+          </div>
+        }
+        {/* Left section aka logo (Page's name, does it count as logo?)*/}
+        <h3 className={styles.left}>
```
Comment (PaulHo0501) [inline components/theme/Navbar.jsx]: Change to "Vườn Hoa Sứ"

File: components/theme/Navbar.jsx [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+          Vườn hoa sứ
+        </h3>
+        {/* Right section */}
+        <div className={styles.right}>
+            {menuSection.map(section => <h5 key={section.id} className={styles.rightItem}>{section.context}</h5>)}
+            <button type='button' className={styles.rightItemPhone} onClick={e => handleClick(e)}>
+              <Image src='/icons/Bars.svg' alt='button' layout='fill' objectFit='contain'/>
```
Comment (PaulHo0501) [inline components/theme/Navbar.jsx]: Please use [react-icons](https://react-icons.github.io/react-icons/)

File: components/theme/Navbar.module.scss [hunk 4]
```diff
@@ -0,0 +1,88 @@
+@use '/styles/variables' as variables;
+
+// Note: 390px
+
+.header {
```
Comment (PaulHo0501) [inline components/theme/Navbar.module.scss:5]: Please make use of mixins and classes

**Review by PaulHo0501 — CHANGES_REQUESTED** [review]

**Comment by YumeAyasaki** [comment]
Update PR:
- Change to Vuon Hoa Su...
- Apply react-icons...
- Edit minor things to match the new global styling...

Note: The same question from the feature/6 + should I update PR with a new comment like this, or reply to your changes requested comment...?

**Comment by PaulHo0501** [comment]
> 

Either is fine

File: components/theme/Navbar.jsx [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+            </div>
+            {menuSection.map(section => 
+            <div className={styles.item} key={section.id}>
+              <div className='sub-header-4'>{section.context}</div>
+            </div>)}
+          </div>
+        }
+        {/* Left section aka logo (Page's name, does it count as logo?)*/}
+        <div className={styles.left}>
+          <div className='sub-header-2'>Vườn Hoa Sứ</div>
```
Comment (PaulHo0501) [inline components/theme/Navbar.jsx]: Please rename class after you rename global styling

**Review by PaulHo0501 — CHANGES_REQUESTED** [review]

**Comment by YumeAyasaki** [comment]
Update PR:
- Update name class...

**Comment by PaulHo0501** [comment]
Please solve all the conflicts

**Comment by PaulHo0501** [comment]
LGTM

**Pull request merged**